go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/free5gc/pcf
 go 1.21
 
 require (
-	github.com/antihax/optional v1.0.0
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/cydev/zero v0.0.0-20160322155811-4a4535dd56e7
-	github.com/free5gc/openapi v1.0.8
+	// github.com/free5gc/openapi v1.0.9-0.20240823100026-6a9f667e914f
+	github.com/free5gc/openapi v1.0.9-0.20241024063921-c9ff412a31a6
```

Comment:
Newest openpai hash: 
`github.com/free5gc/openapi v1.0.9-0.20241024114001-1414ef920de2`
And then you should notice the ue.go in context. (I left a comment)